app/views/idv/gpo/index.html.erb

```diff
@@ -42,4 +42,5 @@
       ],
     ) %>
 
-<%= render 'idv/shared/back', fallback_path: @presenter.fallback_back_path %>
+<% back_or_cancel_params = @presenter.back_or_cancel_parameters %>
+<%= render back_or_cancel_params.delete(:path), back_or_cancel_params %>
```

Comment:
This is a little tricky to follow, what if they were separate methods?

```suggestion
<%= render @presenter.back_or_cancel_partial, @presenter.back_or_cancel_parameters %>
```

Reply:
Done in 6a01b2e